app/scripts/lib/setupSentry.js

```diff
@@ -171,13 +177,13 @@ export const SENTRY_BACKGROUND_STATE = {
     dismissSeedBackUpReminder: true,
     featureFlags: true,
     forgottenPassword: true,
-    identities: false,
     incomingTransactionsPreferences: true,
     ipfsGateway: false,
     isLineaMainnetReleased: true,
     knownMethodData: false,
     ledgerTransportType: true,
     lostIdentities: false,
+    selectedAddress: true,
```

Comment:
Change to false

Reply:
Changed in 8e9da248ef551773ea52922f6359f2c41612a3da